libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -197,15 +189,20 @@ - (void)initializeAppViewState
     self.window.rootViewController = self.viewController;
     [self.window makeKeyAndVisible];
 }
-                                                            
 
-@end
+- (void)logWithLevel:(DDLogLevel)level format:(NSString *)format, ... {
```

Comment:
Ah, I see. So you're suggesting the component name be the same for tests and sample apps. I could create a `default` logger for that - sure.